src/renderer/angular/controllers/commons/dialogs/connecting-to-ledger-controller.js

```diff
@@ -42,7 +42,7 @@ function ConnectingToLedgerController(
 				$rootScope.openChooseLedgerAddressDialog(accounts, ACCOUNTS_QUANTITY_PER_PAGE);
 			})
 			.catch(err => {
-				log.error(err);
+				log.error(err.message);
```

Comment:
This doesn't log the stacktrace. @sk91 `log.error(err)` should work here, right?

Reply:
@dfcowell `log.error(err)` does't works, there is an error while logging the `err` object, 
so next line `onError();` newer be executed.
An issue is: connection to ledger screen is newer be disappeared when ledger is not well connected.

Reply:
@dfcowell Apparently the logging issue is fixed by another commit, so i'm closing this pull request.